loadtest/bulkload.go

```diff
@@ -587,8 +587,29 @@ func LoadPosts(cfg *LoadTestConfig, driverName, dataSource string) {
 			}
 		}
 
+		if _, err := db.Exec("SET autocommit=0,unique_checks=0,foreign_key_checks=0"); err != nil {
+			mlog.Error("Couldn't temporary set values for performance.", mlog.Err(err))
+			return
+		}
+		defer func() {
+			if _, err := db.Exec("SET autocommit=1,unique_checks=1,foreign_key_checks=1"); err != nil {
+				mlog.Critical("Couldn't set temporary values back to defaults, your DB is in a bad state!", mlog.Err(err))
```

Comment:
Unless I'm misunderstanding something here, these settings should only apply to the current session and not to the database as a whole, so I don't think this leaves the database in a bad state. 

Instead of tearing things down in the session, we could open a new session to the database for the purpose of import, set these flags, and then just close it afterwards.

I'm 0/5 on cleaning up vs. new session, but 5/5 on saying `DB is in a bad state!` if it's not actually true.

Reply:
Thanks for clearing up my misunderstanding or how `SET` works!